app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,133 @@
+    subgraph BillingService[Billing Service]
+      direction LR
+      KongGatewayService[Gateway service 1]
+      PagerDutyServiceMapping[PagerDuty service 1]
+      GitHubRepoMapping[GitHub repository 1]
+    end
+
+    %% Define the Inventory Service Catalog Service subgraph inside Konnect Service Catalog
+    subgraph InventoryService[Inventory Service]
+      direction LR
+      SomeOtherService[Gateway service 2]
+      AnotherService[PagerDuty service 2]
+      DifferentRepo[GitHub repository 2]
+    end
+
+  end
+
+  %% Connect the resources to the Konnect Service Catalog
+  GitHub -- "GitHub repositories" --> KonnectServiceCatalog
+  PagerDuty -- "PagerDuty services" --> KonnectServiceCatalog
+  GatewayManager -- "Gateway services" --> KonnectServiceCatalog
+
+  %% Style the subgraphs
+  class KonnectServiceCatalog konnect
+  class BillingService serviceCatalog
+  class InventoryService serviceCatalog
+{% endmermaid %}
+<!-- vale on-->
+
+> Figure 1: This diagram shows how you can use both external integrations, like GitHub and PagerDuty, as well as built-in integrations like Gateway Manager to pull resources into Service Catalog. You can then map those resources (like GitHub repositories, PagerDuty services, and Gateway services) to Service Catalog Services.
+
+## Service Catalog use cases
+
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Keep track of your organization's service ownership by mapping teams to Services. | Add the Service's owner when you [create a new Service in {{site.konnect_short_name}}](https://cloud.konghq.com/service-catalog/create-service) |
+| Gain visibility into all your Services, including unrecognized or undiscovered APIs in your organization.  | [Service Catalog integrations](https://cloud.konghq.com/service-catalog/integrations) |
+| Consolidate key Service health metrics, documentation, and API specs into a single dashboard, allowing you to interact with other tools from one place. | [Service Catalog dashboard](https://cloud.konghq.com/service-catalog/) |
+
+<!-- commenting this out until it's released:
+| Govern how services are created and maintained across your company to adhere to security, compliance, and engineering best practices. | Scorecards |-->
+
+## Service Catalog terminology
+
+| Term | Definition |
+| ---- | ---------- |
+| Integration | These are applications, either {{site.konnect_short_name}}-internal or external, that act as sources from which you can ingest resources. For example, GitHub. |
+| Resource | An umbrella term that denotes entities ingested by Service Catalog from enabled integrations. A resource can range from an infrastructural component (like Gateway services, mesh services, databases, and caches) to an external application or tool (like code repositories, CI/CD infrastructure, and on-call systems) to a piece of documentation (like API specs). Resources can be mapped to one or more Services. |
```

Comment:
@miko-bautista I decided against capitalizing "resources" and "integrations" just because we usually don't capitalize terms/features like that. "Service Catalog Services" is kind of the exception because I wanted to make it clear that a Service Catalog Service isn't the same as a Gateway service.

@Guaris What do you think?

Reply:
Personally, I lean towards capitalizing everything. They are all proper nouns that refer to entities within Service Catalog's object-model. It feels rather strange to capitalize Service, but not integration or resource.

That said, I'll leave the final decision up to you and Angel!

Reply:
@miko-bautista I talked to the other writers, and we thought it would be best to leave them all uncapitalized since other entities aren't capitalized either. 

I revised this page so that any time we're referring to a service in Kong Gateway, I say "Gateway service" and for a service in Service Catalog, I say "Service Catalog service". Just so it's always clear to the user which service we're talking about.